cluster/shard_mapper.go

```diff
@@ -230,10 +230,21 @@ func (r *RemoteMapper) NextChunk() (chunk interface{}, err error) {
 			}
 			aggValues = append(aggValues, v)
 		}
-		mo.Values = []*tsdb.MapperValue{&tsdb.MapperValue{
-			Value: aggValues,
-			Tags:  mvj[0].Tags,
-		}}
+
+		if mvj[0].Time != 0 {
```

Comment:
hi @otoolep, have you checked https://github.com/influxdb/influxdb/issues/4801 already?  let's say if you execute `select mean(value) from mem_used group by time(1m)` ,  `Time` won't be zero because every interval has its own timestamp. however if you execute `select mean(value) from mem_used` without `groupby`, then `Time` will be ignored, and the value of `Time` is 0.